{
  "affected_release": null,
  "package_state": null,
  "threat_severity": "",
  "public_date": "2024-01-03T00:00:00Z",
  "bugzilla": {
    "description": "YASM: remote attacker to cause a denial of service via the yasm_section_bcs_first",
    "id": "2256605",
    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2256605"
  },
  "cvss": {
    "cvss_base_score": "",
    "cvss_scoring_vector": "",
    "status": ""
  },
  "cvss3": {
    "cvss3_base_score": "0.0",
    "cvss3_scoring_vector": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:N",
    "status": "draft"
  },
  "iava": "",
  "cwe": "CWE-400",
  "statement": "Red Hat Product Security does not consider this to be a vulnerability.",
  "acknowledgement": "",
  "name": "CVE-2023-49557",
  "document_distribution": "",
  "details": [
    "No description is available for this CVE."
  ],
  "references": [
    "https://www.cve.org/CVERecord?id=CVE-2023-49557\nhttps://nvd.nist.gov/vuln/detail/CVE-2023-49557\nhttps://github.com/yasm/yasm/issues/253"
  ]
}